unfeaturized slots (#6817)
RasaHQ/rasa: docs/docs/unexpected-input.mdx

      - outdoor_seating
      - preferences
      - feedback
    influence_conversation: true
```

This means that the dialogue model will pay attention to the value of the slot when making a prediction

> (read more about other [featurized slots](./domain.mdx#slots), whereas slots with
> the property `influence_conversation` set to `false` are only used for storing information.

Reviewer: ```suggestion
(read more about how [slots influence the assistant's behaviour](./domain.mdx#slots-and-conversation-behavior)).
```
